packages/prerender/src/__tests__/detectRoutes.test.ts

```diff
@@ -50,18 +50,20 @@ describe('Detecting routes', () => {
     )
   })
 
-  it('Should ignore not found page', () => {
+  it('Should render notFoundPage as 404.html', () => {
     mockedRoutes = [
       {
-        name: 'bazinga',
-        path: '/bazinga',
+        name: undefined,
```

Comment:
This is how structure outputs NotFoundPages

Reply:
would `path: "some-unknown-path"` not be more clearly indicate intent? Or doesn't that work in this case?

Reply:
We're mocking the output of the structure package here. This is just what it outputs, when it comes across a route with a not found prop.

```
      <Route notfound page={NotFoundPage} prerender />
```

The code that deals with that actually doesn't really care about the values of name/path, i've just made this change to more accurately represent what structure does.